skyportal/handlers/api/news_feed.py

```diff
@@ -44,13 +54,27 @@ def get(self):
               application/json:
                 schema: Error
         """
-        preferences = (
-            self.current_user.preferences if self.current_user.preferences else {}
-        )
-        if 'newsFeed' in preferences and 'numItems' in preferences['newsFeed']:
-            n_items = min(int(preferences['newsFeed']['numItems']), 50)
+
+        if hasattr(self.current_user, 'preferences'):
+            preferences = (
+                self.current_user.preferences if self.current_user.preferences else {}
+            )
         else:
-            n_items = 10
+            preferences = {}
+
+        n_items = self.get_query_argument("numItems", None)
+        if n_items is None:
+            if 'newsFeed' in preferences and 'numItems' in preferences['newsFeed']:
+                n_items = min(int(preferences['newsFeed']['numItems']), 50)
```

Comment:
I don't get where `50` comes from.

Reply:
This was the original default

Reply:
Shouldn't this number be either the new default or the new maximum or something that's somehow documented?

Reply:
yeah i am setting this to a default at the top of the file.